tests/yast2_gui/yast2_control_center.pm

```diff
@@ -47,71 +48,83 @@ sub run() {
 
     #	start Media Check
     assert_and_click "yast2_control-center_media-check";
-    assert_and_click "yast2_control-center_media-check_close";
+    assert_screen "yast2_control-center_media-check_close";
+    send_key "alt-l";
     assert_screen 'yast2-control-center-ui';
 
     #	start Online Update
-    assert_and_click "yast2_control-center_online-update";
-    assert_screen "yast2_control-center_update-repo-dialogue";
-    send_key "alt-n";
-    assert_and_click "yast2_control-center_online-update_close";
+    assert_and_click 'yast2_control-center_online-update';
+    assert_screen [qw(yast2_control-center_update-repo-dialogue yast2_control-center_online-update_close)];
+    if (match_has_tag('yast2_control-center_update-repo-dialogue')) {
+        send_key 'alt-n';
+        assert_screen 'yast2_control-center_online-update_close';
+    }
+    send_key 'alt-c';
     assert_screen 'yast2-control-center-ui';
 
     #	start Software Repositiories
     assert_and_click "yast2_control-center_software-repositories";
     assert_screen "yast2_control-center_configured-software-repositories";
-    assert_and_click "yast2_control-center_configured-software-repositories_ok";
+    send_key "alt-o";
     assert_screen 'yast2-control-center-ui';
 
     #	start Printer
     assert_and_click "yast2_control-center_printer";
+    if (check_screen "yast2_control-center_printer_running-cups-daemon") {
+        send_key "alt-n";
+        check_screen "yast2_control-center_printer_running-cups-daemon_error";
+        send_key "alt-o";
+        check_screen "yast2_control-center_detect-printer-queues_error";
+        send_key "alt-o";
+        check_screen "yast2_control-center_show-printer-queues_error";
+        send_key "alt-o";
```

Comment:
because I don't want to let restart cups, Anyway I will check this section and can change this a little bit and see what is better to allow to restart cups locally if delay is not too long.